tests/ci/common_posix_setup.sh

```diff
@@ -23,7 +23,7 @@ PLATFORM=$(uname -m)
 
 NUM_CPU_THREADS=''
 KERNEL_NAME=$(uname -s)
-if [[ "${KERNEL_NAME}" == "Darwin" ]]; then
+if [[ "${KERNEL_NAME}" == "Darwin" || "${KERNEL_NAME}" == "OpenBSD" ]]; then
```

Comment:
This appears to be true for other *BSDs as well, e.g., [FreeBSD](https://man.freebsd.org/cgi/man.cgi?sysctl(8)) and [NetBSD](https://man.netbsd.org/sysctl.8). Could we match instead:
```
if [[ "${KERNEL_NAME}" == "Darwin" || "${KERNEL_NAME}" =~ *BSD ]]; then
```